src/api/api.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable */
+import axios from 'axios';
+
+const rocketsUrl = 'https://api.spacexdata.com/v3/rockets';
+const missionsUrl = 'https://api.spacexdata.com/v3/missions';
+
+const getRockets = async () => await axios.get(`${rocketsUrl}`).then((result) => {
+  const rockets = [];
+  if (result.status === 200) {
+    const { data } = result;
+    data.forEach((rocketData) => {
+      const rocket = {
+        id: rocketData.rocket_id,
+        name: rocketData.rocket_name,
+        description: rocketData.description,
+        flickr_images: rocketData.flickr_images,
```

Comment:
- [ ] **OPTIONAL**
In fetching the flicker images part, I noticed that you're saving the whole images array in the Redux store, then select the first image when rendering 👏 How about receiving the first image of the array directly and it'll be the only image stored in the Redux store! 😄 This could be done by `flicker_image: rocketData.flicker_images[0],` Please tell me what you think? 🙏